fun/draw.cpp

```diff
@@ -129,7 +131,7 @@ int readCmd() {
 
 int main() {
   if (!setupTerm()) {
-    cerr << "term setup error" << endl;
+    fputs("term setup error", stderr);
```

Comment:
The short answer is that <iostream> is an abomination and you should generally avoid it. Overloading "<<" is idiomatic but totally insane. It is (used to be?) explicitly banned in the Google Style guide with the exception of log messages (for esoteric reasons).